guides/0000-atomic-swap.md

```diff
@@ -0,0 +1,97 @@
+### What is atomic swap
```

Comment:
I prefer the template format. It seems more intuitive and easier to follow.

Reply:
I've change the format. What do you think?